releases/release-4.0.13.md

```diff
@@ -0,0 +1,128 @@
+    - Fix resource leak of Shuffle Executor. [#23888](https://github.com/pingcap/tidb/pull/23888)
+    - Fix some string function get wrong result [#23878](https://github.com/pingcap/tidb/pull/23878)
+    - Fix the panic when we calculate the partition range [#23689](https://github.com/pingcap/tidb/pull/23689)
+    - Fix a panic on batch point get for non-partitioned table with partition meta information. When a cluster is upgrade from a old version, its partition meta information maybe not null, but the `Enable` field is false, it should be treat as a non-partitioned table. [#23682](https://github.com/pingcap/tidb/pull/23682)
+    - When TiDB was configured to listen on TCP and UNIX sockets, connections over TCP did not correctly validate that the remote host was permitted to connect. [#23513](https://github.com/pingcap/tidb/pull/23513)
+    - A set of client_errors_summary tables has been added to Information Schema. This helps keep track of which errors have been sent to clients. [#23267](https://github.com/pingcap/tidb/pull/23267)
+    - Fix a bug may cause wrong constant propagation and get the wrong result. [#22923](https://github.com/pingcap/tidb/pull/22923)
+    - Fix wrong bucket name of coprocessor cache. [#22617](https://github.com/pingcap/tidb/pull/22617)
+    - Fix panic occurs when stats inconsistency [#22565](https://github.com/pingcap/tidb/pull/22565)
+
++ TiKV
+
+    - Support back pressure CDC scan speed. [#10145](https://github.com/tikv/tikv/pull/10145)
+    - Fix interference between connections to the same region. [#10144](https://github.com/tikv/tikv/pull/10144)
+    - Cdc: skip seek old value for Put if cache returns None [#10141](https://github.com/tikv/tikv/pull/10141)
+    - Reduce memory usage of CDC initial scan. [#10134](https://github.com/tikv/tikv/pull/10134)
+    - Fix potential panics when input of cast_string_as_time is invalid UTF-8 bytes [#9994](https://github.com/tikv/tikv/pull/9994)
+    - Cdc: limit scan speed (128MB/s by default) [#9983](https://github.com/tikv/tikv/pull/9983)
+    - Fix the bug that TiKV cannot startup when the end of file dict file is damaged. [#9963](https://github.com/tikv/tikv/pull/9963)
+
++ PD
+
+    - Metrics: let tso processing time not include consumption on the network [#3524](https://github.com/pingcap/pd/pull/3524)
+    - Dashboard: update to v2021.03.12.1 [#3469](https://github.com/pingcap/pd/pull/3469)
+
++ Tools
+
+    - BR
+
+        * BR would check cluster version of backup now. [#1090](https://github.com/pingcap/br/pull/1090)
+        * BR now support backing up user tables created in the `mysql` schema. [#1077](https://github.com/pingcap/br/pull/1077)
+        * BR now can tolerate minor TiKV disconnection. [#1062](https://github.com/pingcap/br/pull/1062)
+
+    - TiCDC
+
+        * Fix bug in flow control [#1779](https://github.com/pingcap/ticdc/pull/1779)
```

Comment:
```suggestion
        * Fix the deadlock caused by flow-control when the sorter's input channel has been blocked. [#1779](https://github.com/pingcap/ticdc/pull/1779)
        * Fix the problem that TiKV GC safe point is blocked indefinitely due to TiCDC changefeed checkpoint stagnation. [#1756](https://github.com/pingcap/ticdc/pull/1756)
        * Implement processor flow control to avoid OOM. [#1751](https://github.com/pingcap/ticdc/pull/1751)
         * Revert the update for explicit_defaults_for_timestamp which requires `SUPER` privilege when replicating to MySQL. [#1749](https://github.com/pingcap/ticdc/pull/1749)
        * Add stale temporary files clean-up in Unified Sorter, and forbids sharing sort-dir. [#1741](https://github.com/pingcap/ticdc/pull/1741)
        * Add HTTP handler for failpoint [#1732](https://github.com/pingcap/ticdc/pull/1732)
```